Common/include/option_structure.inl

```diff
@@ -1606,11 +1606,15 @@ class COptionTurboPerformance : public COptionBase {
   unsigned short& size;
   string*& marker_turboIn;
   string*& marker_turboOut;
+  string*& markers;
 
  public:
   COptionTurboPerformance(const string option_field_name, unsigned short& nMarker_TurboPerf,
-                          string*& Marker_TurboBoundIn, string*& Marker_TurboBoundOut)
-      : size(nMarker_TurboPerf), marker_turboIn(Marker_TurboBoundIn), marker_turboOut(Marker_TurboBoundOut) {
+                          string*& Marker_TurboBoundIn, string*& Marker_TurboBoundOut, string*& Marker_Turbomachinery)
+      : size(nMarker_TurboPerf),
+        marker_turboIn(Marker_TurboBoundIn),
+        marker_turboOut(Marker_TurboBoundOut),
+        markers(Marker_Turbomachinery) {
     this->name = option_field_name;
   }
 
```

Comment:
```suggestion
~COptionTurboPerformance() override {
delete [] markers;
delete [] marker_turboIn;
delete [] marker_turboOut;
}
```
Did you try to delete the resources here?

Reply:
Yes tried this in [c5872c4] and [52d8ca9], the code segfaults when trying to finalize the solver when deleting the CConfig class.